techmatters/terraso-mobile-client
# fix: Soil Color Guide matches design + move general info modals to same directory

## Description
In Soil Color Guide:
- Separate out metric vs. imperial strings
- Add the correct Step 5 and move the existing Step 5 to be Step 6
Also: 
- Organize directory structure for info modal files

### Checklist

### Related Issues
Fixes #1115 

### Verification steps
1. See Soil Color Guide and confirm that step "2. Prepare soil sample" says "2mm sieve" and "anything larger than 2mm". There's no way to see imperial units in the app currently, but I promise it would look like this: 
![Color-Propsed-Imperial](https://github.com/techmatters/terraso-mobile-client/assets/5590815/644e7455-02e6-4b57-b400-07568cd921b6)
2. Also in Soil Color Guide, confirm that step "5. Compose photo." exists and matches the [Figma](https://www.figma.com/file/KMJbhjVyrrLFHj6jGcX9lN/LandPKS-Design-Prototype-working-file?type=design&node-id=6168-21844&mode=design&t=oMk90TZVExCQdqPL-0)
